code/__DEFINES/maps.dm

```diff
@@ -43,9 +43,17 @@ Last space-z level = empty
 #define ZLEVEL_CITYOFCOGS 6
 #define ZLEVEL_EMPTY_SPACE 12
 //Unless you modify it in map config should be equal to ZLEVEL_SPACEMAX
-#define ZLEVEL_TRANSIT 13
+#define ZLEVEL_RESERVED 13
+#define ZLEVEL_TRANSIT 13		//Transit is currently as the reserve level.
 
 #define ZLEVEL_SPACEMIN 3
 #define ZLEVEL_SPACEMAX 13
 
-#define SPACERUIN_MAP_EDGE_PAD 15
\ No newline at end of file
+#define SPACERUIN_MAP_EDGE_PAD 15
+
+#define RESERVE_TURF(T)\
```

Comment:
This should be a turf proc

Reply:
isn't it faster like this though it's not like it needs anything from the turf that can't be done in a define?

Reply:
Shuttle code is already laggy and one of the bigger causes of noticeable tick_overrun as it is and is hard to tick_check without issues, (ask ninja) I'd say it's worth it for those reasons to define something like this and avoid proc overhead.

kevin: variables created in macros should be all caps to avoid conflicting, and add them to a `{}` block so they don't conflict with other calls to the same macro within the same scope.

(I can't remember if byond lets you do empty scope blocks, but if not, you can just do if (TRUE) {macro code})